draft-ietf-httpbis-cache-latest.xml

```diff
@@ -331,7 +331,11 @@
   <li><t>The request method is understood by the cache, and</t></li>
   <li><t>the response status code is final (see <xref
   target="associating.response.to.request"/>), and</t></li>
-  <li><t>the response status code is understood by the cache, and</t></li>
+  <li><t>the response status code is understood by the cache, if it is 206 or
+  304 (because these responses need special handling by caches), and</t></li>
```

Comment:
Pre-existing issue:  You're using commas inside a comma-delineated list; the list should be using semicolons.